runtime/service.go

```diff
@@ -564,6 +567,15 @@ func (s *service) createVM(requestCtx context.Context, request *proto.CreateVMRe
 		return errors.Wrapf(err, "failed to start the VM")
 	}
 
+	defer func() {
```

Comment:
How about adding this defer right after calling newJailer()?

Reply:
Hmm, I think I like this at `machine.Start` because that's when the jailer logic would be ran. If we added this after `newJailer` and `machine.Start` wasn't called then this would always error out.

Reply:
I see. I prefer the simplicity of having `defer` right after instantiating an object (like os.Open) rather than remembering which methods are leaving files/processes. But I'm fine with both ways.